manifests/cinder-csi-plugin/csi-provisioner-cinderplugin.yaml

```diff
@@ -47,6 +47,7 @@ spec:
             - /bin/cinder-csi-plugin
             - "--nodeid=$(NODE_ID)"
             - "--endpoint=$(CSI_ENDPOINT)"
+            - "--cluster=kubernetes"
```

Comment:
you want to point it to CO cluster name not on the CO name??
--cluster=<CO-cluster-name>

Reply:
I am using `kubernetes` as the default value just like `--cluster-name string     Default: "kubernetes"` for kube-controller-manager service.

Reply:
ok, can you add it via env?? as other arguments. and also update somewhere in doc, about this param and how to get this

Reply:
Done